include/SFML/Audio/Listener.hpp

```diff
@@ -77,19 +77,19 @@ SFML_AUDIO_API void setGlobalVolume(float volume);
 ///
 /// \return Current global volume, in the range [0, 100]
 ///
-/// \see setGlobalVolume
+/// \see `setGlobalVolume`
 ///
 ////////////////////////////////////////////////////////////
 [[nodiscard]] SFML_AUDIO_API float getGlobalVolume();
 
 ////////////////////////////////////////////////////////////
 /// \brief Set the position of the listener in the scene
 ///
-/// The default listener's position is (0, 0, 0).
+/// The default listener's position is `(0, 0, 0)`.
```

Comment:
I'm not sure if "(0, 0, 0)" counts as code. I suppose it count be interpreted as part of a bigger expression like `sf::Vector3(0, 0, 0)`.